index.html

```diff
@@ -1,299 +1,243 @@
-            I've also been a good helping hand to a lot of developers in
-            different communities!
-            <br />
-            if you'd like to get in contact about potential job opportunities,
-            please reach out!
-          </p>
-          <a
-            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
-            class="link"
-            target="_blank"
-            rel="noreferrer noopener"
-          >
-            Get my resume
-          </a>
-          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
-        </section>
-        <hr />
-        <div class="about-me-section stacks">
-          <ul>
-            <li class="languages list-title">
-              Languages
-              <ul>
-                <li>JavaScript</li>
-                <li>TypeScript</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
-                <li>Sass</li>
-              </ul>
-            </li>
-          </ul>
-          <ul>
-            <li class="frameworks list-title">
-              Frameworks
-              <ul>
-                <li>React & Redux</li>
-                <li>Material-UI</li>
-                <li>Jest</li>
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-              </ul>
-            </li>
-          </ul>
-          <ul>
-            <li class="skills list-title">
-              Skills
-              <ul>
-                <li>GitHub</li>
-                <li>CodePen</li>
-                <li>GitLab</li>
-                <li>Terminal</li>
-                <li>WSL</li>
-                <li>Remote Pair-Programming</li>
-                <li>Teamwork</li>
-                <li>Mentoring</li>
-                <li>Code Reviews</li>
-                <li>Mobile/Responsive Development</li>
-              </ul>
-            </li>
-          </ul>
+    </div>
+    <div class="contact" id="contact">
+      <div class="bg-container">
+        <div class="container">
+          <div class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please
+            get in
+            touch.
+          </div>
+          <form method="post" action="https://formspree.io/f/xeqvrzly">
+            <div class="name-group">
+              <input type="text" name="fname" maxlength="30" placeholder="First Name" required>
+              <input type="text" name="lname" maxlength="30" placeholder="Last Name" required>
+            </div>
+            <input class="email" type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2, 4}$" name="email"
+              placeholder="Email address" required>
+            <textarea name="message" maxlength="500" class="message" rows="5"
+              placeholder="Write your message here" required></textarea>
+            <button type="submit" class="btn">Get in touch</button>
```

Comment:
- [ ] It would be better if we implement the user interactions as pointed out earlier. This user interaction improves user experience and helps attract a large number of people to the site. There are quite a number of CSS attributes for that, for example; `:focus`, `:hover`, `:cursor` ,  `:active`, etc.